Model/Transaction.php

```diff
@@ -162,7 +162,7 @@ protected function buildFromAddress(
             'from_zip' => $fromPostcode,
             'from_state' => $fromState,
             'from_city' => $fromCity,
-            'from_street' => $fromStreet1 . $fromStreet2
+            'from_street' => $fromStreet1 . ', ' . $fromStreet2
```

Comment:
Any need for a check to see if $fromStreet2 actually has a value? I don't think we want to end up with a comma on the end of $fromStreet1 when $fromStreet2 is empty.

Reply:
Good catch @dallendalton!